nifi-commons/nifi-expression-language/src/test/java/org/apache/nifi/attribute/expression/language/TestQuery.java

```diff
@@ -308,10 +311,9 @@ public void implicitDateConversion() {
 
         // the date.toString() above will end up truncating the milliseconds. So remove millis from the Date before
         // formatting it
-        final SimpleDateFormat sdf = new SimpleDateFormat("yyyy/MM/dd HH:mm:ss.SSS", Locale.US);
-        final long millis = date.getTime() % 1000L;
-        final Date roundedToNearestSecond = new Date(date.getTime() - millis);
-        final String formatted = sdf.format(roundedToNearestSecond);
+        final DateTimeFormatter dtf = DateTimeFormatter.ofPattern("yyyy/MM/dd HH:mm:ss.SSS", Locale.US);
+        final Instant roundedToNearestSecond = date.toInstant().truncatedTo(ChronoUnit.SECONDS);
```

Comment:
The variable used to have a wrong name in the original version too because it was truncated, not rounded (which is more obvious with the current implementation).
Could be please fix it and change the name to `truncatedToSecond`?